Dev

andgarzonmal opened the pull request:
- [ ]  Use React documentation.
- [ ]  Use React components.
- [ ]  Use React props.
- [ ] Use React Router.
- [ ]  Connect React and Redux.
- [ ]  Handle events in a React app.
- [ ]  Write integration tests with a React testing library.
- [ ]  Use styles in a React app.
- [ ]  Use React life cycle methods.
- [ ]  Apply React best practices and language style guides in code.
- [ ]  Use store, actions and reducers in React.

andgarzonmal committed:
Add babel

andgarzonmal committed:
add eslint

andgarzonmal committed:
install packages and set up basic react

andgarzonmal committed:
add stylelint

andgarzonmal committed:
add readme

andgarzonmal committed:
add linters

andgarzonmal committed:
add index

andgarzonmal committed:
basic set up project

andgarzonmal committed:
Merge pull request #1 from andgarzonmal/setup

Setup

andgarzonmal committed:
fix linter

andgarzonmal committed:
changue name

andgarzonmal committed:
Add packages

andgarzonmal committed:
set up provider

andgarzonmal committed:
modify index

andgarzonmal committed:
add styles

andgarzonmal committed:
add main component

andgarzonmal committed:
add styles

andgarzonmal committed:
add nav component

andgarzonmal committed:
add configure store reducer

andgarzonmal committed:
add dogs actions, reducers and action creators

andgarzonmal committed:
get data

andgarzonmal committed:
fix linters

andgarzonmal committed:
Merge pull request #2 from andgarzonmal/main-page

Main page

andgarzonmal committed:
Ad styles to App.css

andgarzonmal committed:
Add js to app.js

andgarzonmal committed:
add styles to dog cards

andgarzonmal committed:
add dog card styles

andgarzonmal committed:
add styles to each size segment

andgarzonmal committed:
Add components to each dog size and filter func

andgarzonmal committed:
fix main css

andgarzonmal committed:
add links

andgarzonmal committed:
set styles

andgarzonmal committed:
arrange data

andgarzonmal committed:
fix linters

andgarzonmal committed:
Merge pull request #3 from andgarzonmal/links

Links

andgarzonmal committed:
Add route for links

andgarzonmal committed:
refactor provider

andgarzonmal committed:
create Dog component

andgarzonmal committed:
refactor dogCard component

andgarzonmal committed:
Merge pull request #4 from andgarzonmal/links

Links

andgarzonmal committed:
install packeges

andgarzonmal committed:
Add tests

andgarzonmal committed:
Add tests

andgarzonmal committed:
Add tests

andgarzonmal committed:
Add tests

andgarzonmal committed:
Add tests

andgarzonmal committed:
Add style to index

andgarzonmal committed:
Add styles to dog

andgarzonmal committed:
add style to card

andgarzonmal committed:
add styles to dog size

andgarzonmal committed:
Add link to main

andgarzonmal committed:
Refactor

andgarzonmal committed:
Add test utils

andgarzonmal committed:
fix linters

andgarzonmal committed:
Merge pull request #5 from andgarzonmal/styles

Styles and testing

andgarzonmal committed:
update readme

andgarzonmal committed:
Merge pull request #6 from andgarzonmal/read-me

update readme

andgarzonmal committed:
Update README.md

keroloussamy commented on README.md:
```diff
@@ -0,0 +1,67 @@
+### Setup
+
+Cloning a repository
+
+- On GitHub.com, navigate to the main page of the repository;
+
+- Above the list of files, click "Code" button;
+
+- Copy the URL to clone the repository. 
+
+  - To clone the repository using HTTPS : `https://github.com/ElsonOtake/TVMaze.git`
+
+  - To clone the repository using an SSH key, including a certificate issued by your organization's SSH certificate authority : `git@github.com:ElsonOtake/TVMaze.git`
+
+  - To clone a repository using GitHub CLI : `gh repo clone ElsonOtake/TVMaze`
+
+- Open Terminal;
+
+- Change the current working directory to the location where you want the cloned directory;
+
+- Type `git clone`, and then paste the URL you copied earlier;
+
+- run `npm install` to install all dependencies;
+
+- run `npm run start` to start the application locally.
+
+
+## Authors
+
+👤 **Andrés Garzon**
+
+- GitHub: [@githubhandle](https://github.com/andgarzonmal)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andres-garzon-maldonado-951a2a180/
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
- [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Whoistolu replied:
- Well Resolved 👍🏽

keroloussamy reviewed (changes requested):
Hi @andgarzonmal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add unit tests for several components of the app and some pure functions in Redux code. And add integration tests using React Testing Library.

![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

andgarzonmal committed:
fix tests

andgarzonmal committed:
Merge branch 'dev' of https://github.com/andgarzonmal/dogs-breeds into dev

andgarzonmal committed:
fix test

andgarzonmal commented:
Hello  i added integration tests on my code. Those tests are on the app.test.js file where i mock the user interaction from the main page to the other components (size dogs and selected dog) . Also i added unit tests and checked for several components, i tested Main comonent, App component, Large component, medidium component and small component wich are the main components of my app, also if you check for the coverage of my tests you can see there is a coverage of 95%

![image](https://user-images.githubusercontent.com/95243996/181385147-bbacffae-df0e-4350-8177-7ce7fc6ec09e.png)

andgarzonmal committed:
Update README.md

andgarzonmal committed:
Update README.md

Whoistolu reviewed (approved):
### Hi @andgarzonmal 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
